tests/apollo/util/bft.py

```diff
@@ -52,6 +52,28 @@
     'num_ro_replicas'
 ])
 
+class NetworkTrafficControl():
```

Comment:
have you considered placing this class in https://github.com/vmware/concord-bft/blob/master/tests/apollo/util/bft_network_partitioning.py ?